cli.ts

```diff
@@ -101,9 +119,14 @@ async function updateAction(
   return _write(updates);
 }
 
-function _findImportMap(): string | undefined {
-  return ["./import_map.json", "./deno.json", "./deno.jsonc"]
```

Comment:
I removed the search for "/import_map.json" here because deno doesn't do that

Actually there is a bug in molt about that I'll file it seperatly